### ISSUE TITLE
eval the best model

### ISSUE THREAD

**Issue by xjtusedy** [malllabiisc/EmbedKGQA#14]
How to eval the best model on the test set? Could you please give me the command?

**Comment by apoorvumang** [comment]
Right now please use the existing command to both train and validate the model. I have added a line in the code so that whenever validation accuracy increases it finds and prints the test accuracy. Please pull the latest code.
